test/functional/framework.go

```diff
@@ -80,8 +80,8 @@ type FluentdFunctionalFramework struct {
 }
 
 func init() {
-	maxDuration, _ = time.ParseDuration("60*2m")
-	defaultRetryInterval, _ = time.ParseDuration("1ms")
+	maxDuration, _ = time.ParseDuration("10m")
```

Comment:
FYI: a better way to initialize duration constants is
```
const maxDuration = 60 *time.Minute
```
Not necessary to change for this PR.